charts/ChartConfig.tsx

```diff
@@ -80,6 +80,7 @@ import { populationMap } from "./PopulationMap"
 import { ColorScaleConfigProps } from "./ColorScaleConfig"
 import { countries } from "utils/countries"
 import { DataTableTransform } from "./DataTableTransform"
+import { getErrorMessageRelatedQuestionUrl } from "admin/client/EditorTextTab"
```

Comment:
Don't include `admin/` code in the client-side - that's also why bundlewatch fails.
You can move this method to `charts/` or `utils/` instead.

Reply:
Right, that makes sense, thanks. Let's try that again.